packages/tests/server/interop/links.test.ts

```diff
@@ -58,7 +58,7 @@ test('chainer', async () => {
 
   const result = await observableToPromise(chain).promise;
   expect(result?.context?.response).toBeTruthy();
-  result!.context!.response = '[redacted]' as any;
+  result.context!.response = '[redacted]' as any;
```

Comment:
Roughly speaking, `expect<T>(value: T).toBeTruthy()` returns `T`. Which lets TypeScript know:

1. `result?.context?.response` must be truthy
2. `result`
3. The `!` non-null assertion in `result!` is unnecessary